text/exercise1/cake.md

```diff
@@ -1,25 +1,30 @@
-## Recipice for Chocolate Cake
+## Recipe for Yellow Cake
 
-Yellow cake is delicious!  This recipie is passed down from my great-great-grandparents.
+Yellow cake is delicious!  This recipe is passed down from my great-great-grandparents.
 
-My great-great-granparents lived in Venice. They enjoyed travelling on the canals. 
+My great-great-grandparents lived in Venice. They enjoyed traveling on the canals.
 Did you know Venice became Austrian territory when Napoleon signed the Treaty of Campo 
 Formio on 12 October 1797? I did not.
```

Comment:
While interesting, this history lesson is less pertinent to the recipe. Consider replacing with a story of what occasions your great-great-grandparents made this cake for.